ntbs-service/README.md

```diff
@@ -101,6 +101,23 @@ Ensure you have `azure-cli` installed
 - `az aks install-cli` to install `kubectl`
 - `az aks get-credentials -g NTBS_Development -n ntbs-envs2` to add appropriate credentials to your `~/.kube/config` file
 
+1. NTBS reference data is updated via the geography database, you will need to restore a backup by carrying out the following steps:
+    1. Connect to the `ntbs-ops-dbs.uksouth.cloudapp.azure.com\NTBS` database server in SSMS.
+    1. In the `Object Explorer` panel, right-click on the `int-geography` database and select `Tasks` -> `Back Up...`.
+    1. Click `OK`.
+    1. After some time, you should see a message informing you that the back up has been successful.
+    1. Connect to the `ntbs-ops-dbs.uksouth.cloudapp.azure.com` VM via Remote Desktop (the username and password can be found in the `ntbs-ops-dbs-credentials` secret in Azure) and locate the back up file you just created.
+    1. Zip the file.
+    1. Copy the file to a temporary location on your development machine.
+    1. Unzip the file.
+    1. In SSMS, connect to your SQL Server instance.
+    1. In the `Object Explorer` panel, right-click on `Databases` and select `Restore Database...`.
+    1. Select the `Device` radio button, and add the relevant back up file via the `...` button.
+    1. Change the name of the database from `int-geography` to `geography`.
+    1. Click `OK`.
+    1. You should see a message saying that the database has restored successfully.
+    1. Run the uspUpdateNTBSRefData stored procedure.
+
```

Comment:
I don't think this belongs here (in the "Deployments" section of the readme). In fact, I think we could get away with not adding this at all since we already link to the reporting project readme higher up.

I think we could get away with adding another paragraph to that section, telling the user to run the stored procedure to seed the reference data (and warning them that if they haven't set up a local geography database then they will need to seed their reference data on their own).

Reply:
I've removed this and will start a PR in the reporting project

Reply:
Sorry, I don't think I was clear in the comment above - I meant that we should add something to the paragraph in this file which is referencing the reporting project (not in the reporting project itself). Seeding the reference data in NTBS is clearly an NTBS concern, so it only makes sense to mention it here.